oem/ibm/configurations/bios/enum_attrs.json

```diff
@@ -697,8 +697,8 @@
         "displayName": "Power Supply 0 is functional",
         "dbus": {
             "object_path": "/xyz/openbmc_project/inventory/system/chassis/motherboard/powersupply0",
```

Comment:
May as well change the displayName to be accurate?

Reply:
Also, probably too late to change the attribute_name?  (that's fine)

Reply:
I thought in HostBoot this would mean functional... but I can change to present. 
requires 2 pieces to go up for it to work.

Reply:
How about this,, we do the FW1030 PR as is, and next week with changes to HB, I change the name, and comments in this.   It would be nice to get a driver for the system team to start there testing with.
see: https://gerrit.openbmc.org/c/openbmc/pldm/+/61015

Reply:
works for me